### PR TITLE
Use the stable prefix for MSC3786.

### PR DESCRIPTION
**Pull request by clokep**
Enables the push rule to not push server ACL events.

I'm not sure if there's anything special to do here or just Do It™️.

Part of #13954.

### CODE REVIEW

**Comment by clokep** [comment]
CC @SimonBrandner does it look like I've missed anything here?

File: changelog.d/13997.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Ignore server ACL changes when generating pushes. Implement [MSC3786](https://github.com/matrix-org/matrix-spec-proposals/pull/3786).
```
Comment (SimonBrandner) [inline changelog.d/13997.feature:1]: Perhaps reword this to make it clear this is making an experimental feature stable

**Review by SimonBrandner — APPROVED** [review]
LGTM - thank you! Totally forgot about the MSC...

File: changelog.d/13997.feature (hunk 1 above)
Comment (clokep) [reply changelog.d/13997.feature:1]: It was disabled by default, so I think we can kind of just say that it is now going to be enabled?

**Review by erikjohnston — APPROVED** [review]
Changes make sense. I also can't remember if we need to do anything special when fiddling with these default rules though :/
